racket/collects/racket/cmdline.rkt

```diff
@@ -300,7 +317,7 @@
                           (cadr line))])
             (unless (and (pair? h)
                          (or (string? (car h)) (andmap string? (car h)))
-                         (andmap string? (cdr h)))
+                         (andmap (lambda (x) (or (string? x) (andmap string? x))) (cdr h)))
```

Comment:
`(andmap string? x)` should be changed to `(and (list? x) (andmap string? x))` to avoid a bad error message from the library when a value that is neither string nor list is passed in.